tests/streamlit_ui_tests/streamlit.spec.ts

```diff
@@ -1,9 +1,302 @@
+    
+    await clickButtonAndAwaitResponse(page, mito, { name: 'Pivot' })
+    
+    await checkOpenTaskpane(mito, 'Create Pivot Table test_pivot');
+
+    // Check new empty tab
+    await page.frameLocator('iframe[title="mitosheet\\.streamlit\\.v1\\.spreadsheet\\.my_component"]').getByText('test_pivot', { exact: true }).click();
+    await expect(mito.getByText('test_pivot', { exact: true })).toBeVisible();
+    await expect(mito.getByText('No data in dataframe.')).toBeVisible();
+    
+    // Add a row, column and value
+    await mito.getByText('+ Add').first().click();
+    await mito.getByText('Column1').click();
+    await awaitResponse(page);
+    await mito.getByText('+ Add').nth(1).click();
+    await mito.getByText('Column2').click();
+    await awaitResponse(page);
+    await mito.getByText('+ Add').nth(2).click();
+    await mito.getByText('Column3').click();
+    await awaitResponse(page);
+
+    // Check that the pivot table has been created
+    await expect(mito.getByText('Column3 count 2')).toBeVisible();
+  });
+  
+  test('Open Merge (horizontal)', async ({ page }) => {
+    const mito = await getMitoFrameWithTestCSV(page);
+    await importCSV(page, mito, 'test.csv');
+    
+    await clickButtonAndAwaitResponse(page, mito, { name: '▾ Merge' })
+    await mito.getByText('Merge (horizontal)').click();
+
+    await expect(mito.getByText('Merge Dataframes')).toBeVisible();
+
+    // Check that Column1 exists
+    const ch1 = await getColumnHeaderContainer(mito, 'Column1');
+    await expect(ch1).toBeVisible();
+  });
+
+  test('Open Concat (vertical)', async ({ page }) => {
+    const mito = await getMitoFrameWithTestCSV(page);
+    await importCSV(page, mito, 'test.csv');
+    
+    await clickButtonAndAwaitResponse(page, mito, { name: '▾ Merge' })
+    await mito.getByText('Concat (vertical)').click();
+
+    await expect(mito.getByText('Concatenate Sheet')).toBeVisible();
+
+    // Test that the sheet is empty
+    await expect(mito.getByText('No data in dataframe.')).toBeVisible();
+  });
+  
+  test('Anti Merge (unique)', async ({ page }) => {
+    const mito = await getMitoFrameWithTestCSV(page);
+    await importCSV(page, mito, 'test.csv');
+    
+    await clickButtonAndAwaitResponse(page, mito, { name: '▾ Merge' })
+    await mito.getByText('Anti Merge (unique)').click();
+
+    await expect(mito.getByText('Merge Dataframes')).toBeVisible();
+    await expect(mito.getByText('unique in left')).toBeVisible();
+  });
+
+  test('Graph', async ({ page }) => {
+    const mito = await getMitoFrameWithTestCSV(page);
+    
+    await clickButtonAndAwaitResponse(page, mito, { name: 'Graph' })
+
+    await expect(mito.getByText('Setup Graph')).toBeVisible();
+    await mito.getByTitle('Select columns to graph on the X axis.').getByText('+ Add').click();
+    await mito.locator('.mito-dropdown-item').first().click();
+    await expect(mito.locator('.plotly-graph-div').first()).toBeVisible();
+  });
+
```

Comment:
There are some more tests for the home tab that might be reasonable to add:
1. If there is no data in the tool, then you shouldn't be able to click pivot, merge, etc. 
2. Test that the AI feature opens when the button is clicked
3. Check that the AI button is not displayed on enterprise 
4. Check that the increase/decrease decimal buttons work 
5. Check the formatting select works

Reply:
- [ ] If there is no data in the tool, then you shouldn't be able to click pivot, merge, etc.
- [ ] Test that the AI feature opens when the button is clicked
- [ ] Check that the AI button is not displayed on enterprise
- [ ] Check that the increase/decrease decimal buttons work
- [ ] Check the formatting select works

Reply:
These other two were hard to write, so skipping for now. We'll keep coming back to them!